index.html

```diff
@@ -174,95 +180,122 @@
       </div>
       <section>
         <h2>
-          Syntax
+          Validation
         </h2>
         <p>
-          When URLs are used for payment method identifiers they MUST be
-          absolute URLs including, at most, a <a>scheme</a>, <a>host</a>,
-          <a>port</a> and <a>path</a>. The URL must be a <a>potentially
-          trustworthy URL</a> as defined in the [[SECURE-CONTEXTS]]
-          specification. Identifier URLs MUST have null <a>query</a> and
-          <a>fragment</a> values.
+          The steps to <dfn data-lt="validate|valid">validate a URL-based
+          payment method identifier</dfn> are given by the following algorithm.
+          The algorithm takes a <var>input</var> string as input and returns
+          either true or an error:
         </p>
+        <ol class="algorithm">
+          <li>Let <var>url</var> be the result of running the <a>URL parser</a>
+          on <var>input</var>.
+          </li>
+          <li>If failure, <a>throw</a> a <code>TypeError</code> exception.
+          </li>
+          <li>If running <a>potentially trustworthy URL</a> with <var>url</var>
```

Comment:
Thanks @zkoch for the pointer.
 
As Anne states at the end of #9, the overloading of this URL as an identifier AND a resource could become a real issue. 

@domenic, crazy thought... but what if, to enable payment manifests, a developer instead had to serve the link relationship (instead of bobpay): 

So, `https://merchant.com/checkout`: 

```HTTP
Link: <https://bobpay.com/manifest>; rel="payment-method-manifest"; pmi="https://bobpay.com"
```

Then, so long as `Link` and referenced PMI are same origin, then "https://bobpay.com/manifest" can be fetched. 

This also has the benefit of allowing prefetching of the manifests.